test/index.js

```diff
@@ -35,31 +17,49 @@ const mockProcessStdout = {
 const mockProcessStderr = {
   write: sinon.stub()
 };
+
 const mockConsoleLog = sinon.stub();
 const mockConsoleError = sinon.stub();
 
-const spawnStub = sinon.stub().returns(mockSpawn);
-const browserDriverManager = proxyquire(
-  '../src/browser-driver-manager',
-  {
-    'child_process': {
-      spawn: spawnStub
+const buildId = '90810624976';
```

Comment:
Some missing test cases:

* --help
* --version
* what happens when you supply a version of chrome that doesn't exist
* what happens when you supply a browser we don't support
* what happens when install fails
* what happens when install partially fails (chrome installs but chromedriver fails)
* what happens when you install twice in a row
    * install the same version twice
    * install two different versions
* what happens when `detectBrowserPlatform` returns `undefined`
* what happens if we can't create the `.browser-driver-manager` folder (e.g. permission issues)

Reply:
These should all be covered now, except `help`, which Commander handles. This could be tested in a child process, but it seems like that would be testing Commander internals.